site2/website-next/static/.htaccess

```diff
@@ -7,4 +7,15 @@ RewriteRule "^(zh-CN|zh-TW|ja|ko|fr)/docs/(.+)$" "/docs/$2" [R=301,DPI,L]
 RewriteRule "^docs$" "/docs/" [R=301,DPI,L]
 RewriteRule "^docs/(.+)-incubating/(.+)$" "https://pulsar.staged.apache.org/docs/$1-incubating/$2" [R=301,DPI,L]
 
-RewriteRule "^docs/(.*)develop-binary-protocol(.*)$" "/docs/$1developing-binary-protocol$2" [R=301,DPI,L]
+# Skip if file exists
+RewriteCond %{REQUEST_FILENAME} -f
+RewriteRule ^ - [L]
+
+RewriteRule "^docs/develop-binary-protocol$" "/docs/developing-binary-protocol/" [R=301,DPI,END]
+RewriteRule "^docs/develop-binary-protocol/$" "/docs/developing-binary-protocol/" [R=301,DPI,END]
+RewriteRule "^docs/(.+)/develop-binary-protocol$" "/docs/$1/developing-binary-protocol/" [R=301,DPI,END]
+RewriteRule "^docs/(.+)/develop-binary-protocol/$" "/docs/$1/developing-binary-protocol/" [R=301,DPI,END]
+
+RewriteRule "^api/client/(?!\d)(.*)$" "/api/client/2.10.x/$1" [R=301,DPI,END]
+RewriteRule "^api/admin/(?!\d)(.*)$" "/api/admin/2.10.x/$1" [R=301,DPI,END]
+RewriteRule "^api/pulsar-functions/(?!\d)(.*)$" "/api/pulsar-functions/2.10.x/$1" [R=301,DPI,END]
```

Comment:
@dave2wave, but these redirections should be updated every major release. I tend to see if there're really some requirements for these aliases.

Reply:
Removing it for now. If it turns out to be a real-world requirement, we can add it back.